src/test/java/LoginTests.java

```diff
@@ -3,68 +3,76 @@
-        Assert.assertTrue (driver.findElement ( By.cssSelector ("[class*='DesktopDropDownMenu__AvatarWrapper']") ).isDisplayed ());
-        driver.quit();
+        submit ( );
+        Assert.assertEquals ( getErrorMessage (), "Cannot login\n" + "Empty email or password" );
     }
 
-
     @Test
-    public static void emptyCredentials() {
-
-        WebDriver driver = new ChromeDriver();
-        driver.get("https://deens.com/login");
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-        WebElement loginButton = driver.findElement(By.cssSelector("[data-testid=\"loginSubmit\"]"));
-        loginButton.click();
-        WebElement uiErrorMessage = driver.findElement(By.cssSelector(".ui.error.message"));
-        String errorMessage = uiErrorMessage.getText();
-        Assert.assertEquals(errorMessage, "Cannot login\n" + "Empty email or password");
-        driver.quit();
+
+    public void wrongCredentials () {
+
+        enterEmail ( "ajgirre@gmail.com" );
+        enterPassword ( "hfdgfdkghkfdl" );
+        submit ( );
+        Assert.assertEquals ( getErrorMessage (), "Cannot login\n" + "no user found with email: ajgirre@gmail.com" );
+
 
     }
 
-    @Test
-    public static void wrongCredentials() {
+    public void enterPassword ( String password ) {
+        driver.findElement ( By.cssSelector ( "#password" ) ).sendKeys ( password );
+    }
 
-        WebDriver driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-        driver.get ( "https://deens.com/login" );
-        WebElement emailButton = driver.findElement ( By.cssSelector ("#email"));
-        emailButton.sendKeys ( "ajgirre@gmail.com" );
-        WebElement incorrectpasswordButton = driver.findElement ( By.cssSelector ("#password"));
-        incorrectpasswordButton.sendKeys ( "jhhfjdkfhd" );
-        WebElement loginButton = driver.findElement (By.cssSelector ("[data-testid=\"loginSubmit\"]"));
-        loginButton.click ( );
-        WebElement uiErrorMessage = driver.findElement ( By.cssSelector (".ui.error.message"));
-        String errorMessage = uiErrorMessage.getText ();
-        Assert.assertEquals ( errorMessage, "Cannot login\n" + "no user found with email: ajgirre@gmail.com" );
-        driver.quit();
+
+    public void enterEmail ( String email ) {
+        driver.findElement ( By.cssSelector ( "#email" ) ).sendKeys ( email );
+    }
 
 
+    public void submit () {
+        driver.findElement ( By.cssSelector ( "[data-testid=\"loginSubmit\"]" ) ).click ( );
     }
+
+    public boolean avatarAppeared () {
+
+        return driver.findElement ( By.cssSelector ( "[class*='DesktopDropDownMenu__AvatarWrapper']" ) ).isDisplayed ( );
+    }
+
+    private String getErrorMessage () {
+        WebElement uiErrorMessage = findElement ( ".ui.error.message" );
+        String errorMessage = uiErrorMessage.getText ( );
+        return errorMessage;
```

Comment:
you use uiErrorMessage only ones. it is not necessary to init a new variable loginField
you might write this command in one line:
return driver.findElement(By.cssSelector("[class='ui message']")).getText();